### PR TITLE
Better err handling

### PR DESCRIPTION
**Pull request by dwalintukan**
# Pull Request Template

## Checklist

- [x] Added CHANGELOG.md entry under Current section.
- [x] If I added new functionality, I added tests covering it.
- [x] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
- [x] I checked whether I should update the docs and did so if necessary.

## Notes

Will handle the regression tests in a later PR.

### CODE REVIEW

File: docker/mainnet/docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
   bodhi-server-mainnet:
     hostname: bodhi-server-mainnet
     container_name: bodhi-server-mainnet
-    restart: unless-stopped
+    restart: always
```
Comment (BernardHan) [inline docker/mainnet/docker-compose.yml]: Remind me why we had `unless-stopped` at the first place?

File: src/index.js [hunk 2]
```diff
@@ -15,10 +20,9 @@ const start = async () => {
     initSync();
     require('./web3');
     initApi();
-    startSync(true);
+    await startSync(true);
```
Comment (BernardHan) [inline src/index.js]: is `await` here a must?

File: docker/mainnet/docker-compose.yml (hunk 1 above)
Comment (dwalintukan) [reply docker/mainnet/docker-compose.yml]: ya it wasnt the proper restart policy to begin with. generally we want `always` in all cases to have max uptime.

File: src/index.js (hunk 2 above)
Comment (dwalintukan) [reply src/index.js]: i was playing around with getting it to handle unhandled promise exceptions, but found a diff solution for it. but either way its proper to set await on it. if it was returning then we wouldn't have to add the await.

**Pull request merged**